.semgrep.yml

```diff
@@ -268,7 +268,7 @@ rules:
       include:
         - aws/
     patterns:
-      - pattern-inside: func $READMETHOD(...) $ERRORTYPE { ... }
+      - pattern-inside: func $FUNCNAME(...) $ERRORTYPE { ... }
```

Comment:
just changed to reflect we're also looking inside `dataSource..Read` and `flatten..Resource` methods as well

Reply:
on second thought, the rule misses looking at the data-source logic 😮